Specs/Scene/ModelExperimental/CustomShaderStageSpec.js

```diff
@@ -0,0 +1,804 @@
+import {
+  AttributeType,
+  CustomShader,
+  CustomShaderStage,
+  LightingModel,
+  ModelAlphaOptions,
+  ModelLightingOptions,
+  ShaderBuilder,
+  UniformType,
+  VaryingType,
+  _shadersCustomShaderStageVS,
+  _shadersCustomShaderStageFS,
```

Comment:
Huh this is the first place I've seen a spec include a shader... but I guess it works because I see it's handled in the gulpfile.